src/data/cassandra/impl/ExecutionStrategy.h

```diff
@@ -230,6 +260,7 @@ class DefaultExecutionStrategy
         auto const numStatements = statements.size();
         std::optional<FutureWithCallbackType> future;
 
+        counters_->registerReadStarted(numStatements);
```

Comment:
I'd prefer to put this on line 262 and add a spacer between the comment - like you do in the next read function :)